projects/plugins/jetpack/_inc/client/at-a-glance/index.jsx

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
```

Comment:
`Fragment` doesn't seem to be used. I suggest removing it, or otherwise using it instead of `< />` in the `<Section />`

Reply:
I think it got added automatically when I used <> and that's why linting is not complaining about it not being used

Reply:
yeah, IDEs :-/

Reply:
not the ide, but the linting command on pre-commit

Reply:
Don't know. just pushed 0cfa1ff7196d3828d29f0d59716db6f260a59ed7 and nothing complained either